### DISCUSSION TITLE
service_code in Allowed Amounts file

### DISCUSSION THREAD

**Discussion by perbs** [CMSgov/price-transparency-guide#229] [Q&A]
I have a question about the meaning of the service_code object in the Allowed Amounts file - where exactly does it fit in in terms of meaning? In the Negotiated Rates file, it lives with the pricing info, so its meaning as the POS types that price applies to is obvious. But in the Allowed Amounts file, it lives with the TIN, separate from any pricing.

So for the Allowed Amounts file, does the service_code represent all the POS codes used by a particular TIN, or is it a grouper for the various amounts attributed to each TIN? If the latter, it seems like it would make sense to move it down alongside pricing information.

**Comment by perbs** [comment]
The more I think about this, the more I wonder if this was done intentionally to help with obfuscating patient data. By aggregating all the applicable service_code values at the TIN level rather than the price level, you can't tell which POS led to which price.

@shaselton-usds  Could you speak to this?

**Reply by frenchcanuk** [reply]
I think that is why we only output allowed amount with more than 20 occurence to protect privacy
